src/App.js

```diff
@@ -37,7 +37,9 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      {tweetsArray.map((tweet) => {
+        return <Tweet tweet={tweet} />;
```

Comment:
keep in mind that we we use a map function react expect to add a key prop to the component in order to identify which component you are refering, in general we have and id when we are rendering data. in this case we dont have an id so we can do something in this case like this is use the name of the user ONLY for this case Good job 🚀

```suggestion
        return <Tweet tweet={tweet}  key={tweet.user.name}/>;
```